### DISCUSSION TITLE
[@types/archiver] version 5.3.0

### DISCUSSION THREAD

**Discussion by A13k2** [DefinitelyTyped/DefinitelyTyped#58021] [Issues with a @types package]
`yarn upgrade-interactive` shows that there is a new version available for `@types/archiver` (5.3.0).

After upgrading it shows that 5.1.1 is the newest version. After this 5.3.0 again.

npm says 5.1.1 is the newest version: https://www.npmjs.com/package/@types/archiver

I got no idea what is happening here...

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
#58024 5.1 is the latest version published to DT, it was not updated on API change recently. I've created PR to bump version to 5.3
